test/common/network/address_impl_test.cc

```diff
@@ -331,6 +360,26 @@ TEST(PipeInstanceTest, Basic) {
   EXPECT_EQ(nullptr, address.envoyInternalAddress());
 }
 
+TEST(InternalInstanceTest, Hashable) {
+  EXPECT_TRUE(absl::VerifyTypeImplementsAbslHashCorrectly({
```

Comment:
You should have a test that calls this on all types of Addresses simultaneously instead of grouping them by type.

Reply:
got it, will add